### DISCUSSION TITLE
Wazuh Email Config not working

### DISCUSSION THREAD

**Discussion by pkgwillie** [Security-Onion-Solutions/securityonion#4804] [Unsupported Versions]
Want to be alerted via email for Wazuh alerts shown on the SO dashboard (v.2.3.52).   What is the log_alert_level and email_alert_level to set in Wazuh config to receive via email the alerts shown in SO dashboard ? Below are the relevant sections of my config, but emails still don't get sent. Would appreciate any steps I can take to troubleshoot to get things working.

/nsm/wazuh/etc/ossec.conf
-----------------------------------------
`<ossec_config>
  <global>
    <jsonout_output>yes</jsonout_output>
    <alerts_log>yes</alerts_log>
    <logall>yes</logall>
    <logall_json>yes</logall_json>
    <email_notification>yes</email_notification>
    <smtp_server>x.x.x.x</smtp_server>
    <email_from>xx@xx.com</email_from>
    <email_to>xx@xx.com</email_to>
    <email_maxperhour>18</email_maxperhour>
  </global>`

 
` <alerts>
    <log_alert_level>1</log_alert_level>
    <email_alert_level>7</email_alert_level>
  </alerts>`

**Comment by DinhDiesel** [comment]
I did the same thing you did, I actually received a notification that stated Ossec started. But I i'm not seeing email alerts after.....not sure if you've gotten this far or actually resolved it?

**Reply by pkgwillie** [reply]
Actually got this resolved. Config was ok. Please refer to similar issue below.
https://github.com/Security-Onion-Solutions/securityonion/discussions/2874#discussioncomment-1015693
